controllers/core/openstackcontrolplane_controller.go

```diff
@@ -140,6 +149,7 @@ func (r *OpenStackControlPlaneReconciler) SetupWithManager(mgr ctrl.Manager) err
 		For(&corev1beta1.OpenStackControlPlane{}).
 		Owns(&mariadbv1.MariaDB{}).
 		Owns(&keystonev1.KeystoneAPI{}).
+		Owns(&placementv1.PlacementAPI{}).
```

Comment:
Just out of curiosity does this line means
 - a) the openstack-operator controller watches the PlacementAPI CRs *it creates*
 - b) the openstack-operator controller watches *all* PlacementAPI CR creation even if it is created manually outside of the openstack-operator controller.

Reply:
It would be `b`.  But it is possible to only have it watch specific `PlacementAPI` CRs, if they are properly labeled and then filtered through a watch function, like we do for ConfigMaps here: https://github.com/openstack-k8s-operators/cinder-operator/blob/77b8278c88f1327ff4ccc050f360a33d246b4c47/controllers/cinderapi_controller.go#L199-L247

Reply:
Thanks @abays